aws/aws-sdk-php-symfony/1.3/config/packages/aws.yaml

```diff
@@ -0,0 +1,10 @@
+# Basic configuration for AWS SDK
+# Please visit http://docs.aws.amazon.com/aws-sdk-php/v3/guide/guide/configuration.html
+# for more information
```

Comment:
For the comment, can you get some inspiration from existing comments in for other packages (just that they are all consistent?).

Reply:
Thanks you. I think I should simply remove it, like the other packages for consistent.